src/server/index.tsx

```diff
@@ -58,19 +58,24 @@ server.get("/robots.txt", async (_req, res) => {
   res.send(robotstxt);
 });
 
-server.get("/css/themes/:name", async (req, res) => {
+server.get("/css/themes/:set?/:name", async (req, res) => {
```

Comment:
This is probably the most controversial change but it allows us to import nicely from themes, i.e. `@import "hljs/default.css"`

Reply:
I'd prefer it be hljs-dark.css , and hljs-light.css, and have them loaded in `theme.tsx` if possible. I'm not sure if symlinking is possible there, but its worth a try. If not we could always have a job to copy the theme from node_modules into the assets/css/themes folder.